cmd/puppet-db/root.go

```diff
@@ -138,24 +144,90 @@ func setCmdFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringP(
 		"cert",
 		"",
-		getDefaultCert(),
+		"",
 		"`Path` to client certificate for auth",
 	)
 	cmd.PersistentFlags().StringP(
 		"key",
 		"",
-		getDefaultKey(),
+		"",
 		"`Path` to client certificate for auth",
 	)
 
 	cmd.PersistentFlags().StringP(
 		"token",
 		"",
-		getDefaultToken(),
+		"",
 		"`Path` to RBAC token for auth (PE Only)",
 	)
 }
 
+func initConfig(cfgFile string) {
+	viper.SetConfigType("json")
+	err := readConfigFile(cfgFile)
+	if err != nil {
+		log.Error(err.Error())
+		os.Exit(1)
+	}
+}
+
+func readConfigFile(cfgFile string) error {
+	if cfgFile != "" {
+		return readCommandLineConfigFile(cfgFile)
+	}
+
+	err := readGlobalConfigFile()
+	if err != nil {
+		return err
+	}
+	return mergeUserConfigFile()
```

Comment:
this logic should be changed to be the same as `rust` implementation:
- read global config file
- read flag config file (flag config value should use default value of `$HOME/.puppetlabs/client-tools/puppetdb.conf`)